Varonis/varonis-data-security/_meta/manifest.yml

```
@@ -0,0 +1,9 @@
+uuid: 7b75d498-4a65-4d44-aa81-31090d723a60
+name: varonis-data-security
+slug: varonis-data-security
+
+description: >-
+  Duo Security offers solutions to track and protect data. This intake ingest alerts from the 
+  Varonis Data Security platform.
+
+data_sources:
```

From [the datasources document](https://github.com/SEKOIA-IO/intake-formats/blob/main/doc/datasources.md):
```suggestion
data_sources:
   Data loss prevention: Varonis detects data exportation
```